src/tests/functional/plugin/shared/src/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -96,6 +112,26 @@ TEST_P(OVPropertiesDefaultTests, CheckDefaultValues) {
     }
 }
 
+TEST_P(OVSetPropComplieModleGetPropTests, SetPropertyComplieModelGetProperty) {
+    OV_ASSERT_NO_THROW(core->set_property(device_name, properties));
+
+    ov::CompiledModel exeNetWork;
+    OV_ASSERT_NO_THROW(exeNetWork = core->compile_model(model, device_name, compileModelProperties));
+
+    for (const auto& property_item : compileModelProperties) {
+        Any exeNetProperty;
+        OV_ASSERT_NO_THROW(exeNetProperty = exeNetWork.get_property(property_item.first));
+        ASSERT_EQ(property_item.second.as<std::string>(), exeNetProperty.as<std::string>());
```

Comment:
what about checking property correctness directly following below code?
`ASSERT_EQ(property_item.second, exeNetProperty);`

Reply:
when checking property ov::hint::allow_auto_batching, ASSERT_EQ(property_item.second, exeNetProperty) will fail. test result: Expected equality of these values:
  value
    Which is: YES
  exeNetProperty
    Which is: YES